OpenRA.Mods.Common/UtilityCommands/CheckMissingSprites.cs

```diff
@@ -38,29 +38,21 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			// any tilesets from being checked further.
 			try
 			{
-				// DefaultSequences is a dictionary of tileset: SequenceProvider
-				// so we can also use this to key our tileset checks
-				foreach (var kv in modData.DefaultSequences)
+				foreach (var kv in modData.DefaultTerrainInfo)
 				{
 					try
 					{
 						Console.WriteLine("Tileset: " + kv.Key);
-						var terrainInfo = modData.DefaultTerrainInfo[kv.Key];
-
-						if (terrainInfo is ITemplatedTerrainInfo templatedTerrainInfo)
+						if (kv.Value is ITemplatedTerrainInfo templatedTerrainInfo)
 							foreach (var r in modData.DefaultRules.Actors[SystemActors.World].TraitInfos<ITiledTerrainRendererInfo>())
```

Comment:
Can `r` be changed to `rule` for clarity? I'm guessing that is what it stands for.